tests/tests_main.cpp

```diff
@@ -362,11 +362,15 @@ void test_base::SetUp() {
   irs::file_utils::mkdir(test_dir_.c_str(), false);
 }
 
-// -----------------------------------------------------------------------------
-// --SECTION--                                                              main
-// -----------------------------------------------------------------------------
+static void AssertCallback(std::string_view file, std::size_t line,
+                           std::string_view function,
+                           std::string_view condition) noexcept {
+  FAIL() << file << ":" << line << ": " << function << ": Condition '"
+         << condition << "' is true.";
```

Comment:
maybe `is false`?

Reply:
No, true, because LOG_MESSAGE(condition) condition should be true
assert add `!`

Reply:
Then we could supply condition as string as separate macros arg and avoid this  "!<condition> is true"
Like this:
#define ASSERT_PRINTER(cond, cond_text)                           \
  do {                                                           \
    if (!(cond)) {                                              \
      std::cout <<  __FILE__ << ", " << __LINE__ << "[" << cond_text << "] is false" << std::endl;     \
    }                                                            \
  } while (false)

#define IRS_ASSERT(cond) ASSERT_PRINTER(cond, #cond)

Reply:
I don't think formatting in macro is good idea, anyway fixed